setup.py

```diff
@@ -125,7 +125,7 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=metadata.get('install_requires', 'astropy').strip().split(),
+      install_requires=eval(metadata.get('install_requires', "['astropy']")),
```

Comment:
I strongly feel that we shouldn't use eval. If this is the only way to parse it, I would vote for using the simple comma-separated format rather than a Python list.